kettle/deployment.yaml

```diff
@@ -23,13 +27,18 @@ spec:
       serviceAccountName: kettle
       containers:
       - name: kettle
-        image: gcr.io/k8s-testimages/kettle:latest
+        image: gcr.io/k8s-staging-infra-tools/kettle:latest
         imagePullPolicy: Always
         env:
         - name: DEPLOYMENT
           value: prod
         - name: SUBSCRIPTION_PATH
           value: kubernetes-jenkins/gcs-changes/kettle-filtered
+        resources:
+          requests:
+            memory: 4Gi
+          limits:
+            memory: 12Gi
```

Comment:
we have no pool big enough to handle this. 🤞🏾 NAP on the `aaa` will auto-provision the nodes needed.

Reply:
Looks like we needed it already before i made this change. 

```
davanum@cloudshell:~/test-infra/kettle (k8s-staging-publishing-bot)$ kubectl get nodes
NAME                                                  STATUS   ROLES    AGE   VERSION
gke-aaa-nap-e2-highmem-4-79yq2sly-10de5ce3-hcki       Ready    <none>   21d   v1.26.14-gke.1006000
gke-aaa-nap-e2-highmem-4-79yq2sly-8a2e99d8-agyz       Ready    <none>   21d   v1.26.14-gke.1006000
gke-aaa-nap-e2-highmem-4-79yq2sly-8a2e99d8-v3eh       Ready    <none>   21d   v1.26.14-gke.1006000
gke-aaa-nap-e2-highmem-4-79yq2sly-ad548c1d-jkur       Ready    <none>   21d   v1.26.14-gke.1006000
gke-aaa-pool1-20190918234853194900000-411e5408-zaxm   Ready    <none>   21h   v1.26.14-gke.1006000
gke-aaa-pool1-20190918234853194900000-8bfa2747-g0cm   Ready    <none>   21d   v1.26.14-gke.1006000
gke-aaa-pool1-20190918234853194900000-be551715-8fg9   Ready    <none>   21d   v1.26.14-gke.1006000
gke-aaa-pool2-20200316195138785800000-91796c71-wyhc   Ready    <none>   21d   v1.26.14-gke.1006000
gke-aaa-pool2-20200316195138785800000-efbc2684-7dsh   Ready    <none>   21d   v1.26.14-gke.1006000
gke-aaa-pool2-20200316195138785800000-fafa121a-6coa   Ready    <none>   21d   v1.26.14-gke.1006000
```